app/scripts/lib/auth-errors.js

```diff
@@ -84,6 +84,10 @@ function (_, Errors) {
       errno: 114,
       message: t('Attempt limit exceeded')
     },
+    ACCOUNT_LOCKED: {
+      errno: 121,
+      message: t('Your account has been locked for security reasons')
+    },
     SERVER_BUSY: {
```

Comment:
We are also going to need a `122` - `Account is not locked` See https://github.com/mozilla/fxa-auth-server/pull/867/files#diff-0f426376b964e609c5c4618eb8bed3b2R599